File: editorial_ch/chap6-16_20/prob6-18.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		// 輸入
		Scanner sc = new Scanner(System.in);
		long A = sc.nextLong();
		long B = sc.nextLong();
		
		// 判斷最小公倍數是否超過 10^18
		if (A / GCD(A, B) > 1000000000000000000L / B) {
			System.out.println("Large");
		}
		else {
			System.out.println(A / GCD(A, B) * B);
		}
	}
	static long GCD(long A, long B) {
		if (B == 0) return A;
		return GCD(B, A % B);
	}
}

File: editorial_ch/chap6-16_20/prob6-18.py
# 使用輾轉相除法求出最大公因數（也可以使用標準函式庫 math.gcd(A, B) 作為其他方法）
def GCD(A, B):
	if B == 0:
		return A
	return GCD(B, A % B)

# 輸入
A, B = map(int, input().split())

# 求出最小公倍數（注：在 Python 中不需在意溢出）
L = A * B // GCD(A, B)
if L > 10 ** 18:
	print("Large")
else:
	print(L)